### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by itsvinayak**
# Welcome to Javascript community

### **Describe your change:**

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Javascript files are placed inside an existing directory.
* [ ] All filenames should use the UpperCamelCase (PascalCase) style.  There should be no spaces in filenames.
     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
> All filenames follow camel casing style.

Is that `UpperCamelCase.js` style or `lowerCamelCase.js` style?  Definately provide an example.  There should be no spaces in file names.  I most like
https://google.github.io/styleguide/jsguide.html#file-name but you can decide.

**Comment by cclauss** [comment]
https://www.robinwieruch.de/javascript-naming-conventions advocates `kebab-case` for backend (algorithms) filenames.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -11,8 +11,9 @@
 * [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
 * [ ] This pull request is all my own work -- I have not plagiarized.
 * [ ] I know that pull requests will not be merged if they fail the automated tests.
-* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
+* [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
 * [ ] All new Javascript files are placed inside an existing directory.
-* [ ] All filenames follow camel casing style.
+* [ ] All filenames follow kebab-case style.There should be no spaces in filenames.<br/>
```
Comment (itsvinayak) [inline .github/pull_request_template.md:16]: Is this ok, @cclauss

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -11,8 +11,9 @@
...
+     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
```
Comment (itsvinayak) [inline .github/pull_request_template.md:17]: is this ok, @cclauss

Comment (cclauss) [reply .github/pull_request_template.md]: https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/ ;-)

File: .github/pull_request_template.md (hunk 1 above)
Comment (cclauss) [inline .github/pull_request_template.md]:
```suggestion
* [ ] All filenames should use the UpperCamelCase style.  There should be no spaces in filenames.<br/>
```

**Pull request merged**